sys/net/gnrc/network_layer/ndp/internal/gnrc_ndp_internal.c

```diff
@@ -311,6 +311,145 @@ void gnrc_ndp_internal_send_rtr_sol(kernel_pid_t iface, ipv6_addr_t *dst)
     gnrc_netapi_send(gnrc_ipv6_pid, hdr);
 }
 
+#if (defined(MODULE_GNRC_NDP_ROUTER) || defined(MODULE_GNRC_SIXLOWPAN_ND_ROUTER))
+static bool _pio_from_iface_addr(gnrc_pktsnip_t **res, gnrc_ipv6_netif_addr_t *addr,
+                                 gnrc_pktsnip_t *next)
+{
+    if (((addr->prefix_len - 1) > 127) && /* prefix_len < 1 || > 128 */
```

Comment:
I chosed to declare the constants just to be unsigned (with the `U` suffix). This should also do the trick. ;-)